README.md

```diff
@@ -1,211 +1,19 @@
-Mattermost Enterprise Edition Helm Chart
-====================================================
+# Mattermost Helm Charts ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-helm/master.svg)
 
-This is the Helm chart for the Mattermost Enterprise Edition. It is subject to changes, but is used by Mattermost internally to run CI servers and our [community.mattermost.com](https://community.mattermost.com) server. To learn more about Helm charts, [see the Helm docs](https://helm.sh/docs/).
+This repo collects a set of [Helm](https://helm.sh) charts curated by [mattermost](https://www.mattermost.com).
```

Comment:
Can we add a list of links to the READMEs of all the included helm charts?

Reply:
yes, will add just for MM-ee for now.
push does not exist yet and mm-te is not merged

Reply:
done